src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,76 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val deleteBackupLocator = By.xpath(
+        "//*[@id='tbl_services']" +
+            "//*[contains(text(), 'com.atlassian.jira.service.services.export.ExportService')]" +
+            "/ancestor::tr" +
+            "//*[contains(@class, 'operations-list')]" +
+            "//*[contains(text(), 'Delete')]"
+    )
+
+    /**
+     * By default, this function will try to delete 10 backup services.
```

Comment:
Where did the "limit 10" come from?
We're deleting backup services for a reason, why would it be ok to leave *some* behind?

Reply:
Leaving some backup services is not ok. That is why there is another function in the API.
Why do we even need such a limit?
To avoid potentials problems with never-ending loop.

Reply:
If it's not ok, then this method is not ok and we need to remove it.
If infinite loops are real then the other method is not ok and we need to remove that.
Let's be consistent here.

Also, suppose the infinite loop is real - then it means the deletions don't work. So we give up after 10 attempts and carry on? The user asked to delete backup services and we didn't do it and silently ignored that failure.
Let's just remove this weird complication. If infinite loops will happen, then timeouts will trip up.